apps/explorer/src/ui/SenderRecipient.tsx

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Amount } from '~/ui/Amount';
+import { Heading } from '~/ui/Heading';
+import { SenderRecipientAddress } from '~/ui/SenderRecipientAddress';
+
+type Recipient = {
+    address: string;
+    coin?: {
+        amount: number;
+        coinSymbol?: string;
+    };
+};
+
+export interface SenderRecipientProps {
+    sender: string;
+    transferCoin?: boolean;
+    recipients?: Recipient[];
+}
+
+export function SenderRecipient({
+    sender,
+    recipients,
+    transferCoin,
+}: SenderRecipientProps) {
+    const multipleRecipients = !!(recipients?.length && recipients?.length > 1);
```

Comment:
You shouldn't need the defined check or the double negation, `recipients?.length > 1` should be plenty.